src/components/TopBar/TopBar.test.js

```diff
@@ -54,7 +54,7 @@ it('renders', () => {
                     <TopBar
                         appName="Demo"
                         appColor="#808080"
-                        appLogo={PowsyblLogo}
+                        appLogo={<PowsyblLogo />}
```

Comment:
But that give a rebuild of the logo at each evaluation no?

Reply:
I'm not sure, maybe, but that's how it works. 
https://github.com/gridsuite/commons-ui/blob/b49db7c8704b5cc18fe239bde6a5536b0d31aa1e/src/components/TopBar/GridLogo.jsx#L36

Reply:
https://github.com/gridsuite/gridstudy-app/blob/8aca6bf551b705fd4413ca525da77778abe74138/src/components/app-top-bar.jsx#L291-L297